Exoft.Gamification.Api/Startup.cs

```diff
@@ -52,14 +54,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddCors();
             services.AddMvc(options =>
```

Comment:
if that's a core 3.1 - maybe it's better to use endpoint routing, MapControllers etc.?